app/lib/account/backend.dart

```diff
@@ -177,7 +178,16 @@ class AccountBackend {
     if (auth == null) {
       return null;
     }
-    return await _lookupOrCreateUserByOauthUserId(auth);
+    final user = await _lookupOrCreateUserByOauthUserId(auth);
+    if (user.isDeleted) {
```

Comment:
If this happens, it's likely because the account was deleted and then recovered. In this case, I think we should change `isDeleted` to true. Hmm... I'm not sure... we should think this through, if we get it wrong we could introduce a security vulnerability.

Reply:
I wouldn't set `isDeleted` to `false` automatically (if that's what you've meant with true :) ), rather let the user escalate the issue. This should be rare, it should be a very exceptional case, we should rather spend a bit of time with it instead automatically give it a pass...